sql/expression_value.h

```diff
@@ -1401,7 +1401,7 @@ struct AnyValueInfo: public ExpressionValueInfoT<ExpressionValue> {
 /// For an embedding
 struct EmbeddingValueInfo: public ExpressionValueInfoT<ML::distribution<CellValue, std::vector<CellValue> > > {
     EmbeddingValueInfo(StorageType storageType = ST_ATOM)
-        : EmbeddingValueInfo({-1}, storageType)
+        : EmbeddingValueInfo(std::vector<ssize_t>({-1}), storageType)
```

Comment:
ssize_t is signed (size_t is unsigned, ssize_t is short for "signed size_t".  Here, it signifies "I don't know the size of that dimension".